tests/unit/conftest.py

```diff
@@ -14,6 +16,8 @@ def create_parser(parser_class, func_to_be_mocked, parsed_output, headers=None):
             )
         }
         FakeApp.configure_mock(**attrs)
+        if props_conf is not None:
+            FakeApp.props_conf.return_value = props_conf
```

Comment:
no need for dedicated `if` for props_conf, you can set return value by passing it as parsed_output:
`parser(pp, "props_conf", _your_props_conf_)`

Reply:
see below comment